dockerhub/travis-ci-image.dockerfile

```diff
@@ -32,14 +28,12 @@ RUN apt-get update -y && apt-get install -qy \
 # For Ceph
         librados-dev \
         liberasurecode-dev \
-        python-rados \
+        python3-rados \
         ceph \
 # For prometheus
         libsnappy-dev \
         libprotobuf-dev
 
-# Install Redis from more recent version of Ubuntu to have >= 3.2 and not only 3.0
-RUN echo 'deb http://old-releases.ubuntu.com/ubuntu artful universe' > /etc/apt/sources.list
 RUN apt-get update -y && apt-get install -qy \
```

Comment:
This should be merged with the `apt-get update/install` above to make the build faster. No reason to have it split anymore.